ReactNativeClient/lib/reducer.js

```diff
@@ -104,6 +105,20 @@ stateUtils.lastSelectedNoteIds = function(state) {
 	return output ? output : [];
 };
 
+stateUtils.getLastSeenNote = function(state) {
+	const selectedNoteIds = state.selectedNoteIds;
+	const notes = state.notes;
+	if (selectedNoteIds != null && selectedNoteIds.length>0) {
+		const currNote = notes.find(note => note.id === selectedNoteIds[0]);
+		if (currNote != null) {
+			return {
+				id: currNote.id,
+				parent_id: currNote.parent_id,
+			};
+		}
+	}
+};
```

Comment:
Missing explicit return for when no note found

Reply:
Good catch, and that makes me think I'll add an eslint rule for explicit returns as I believe there's one.

Reply:
Sounds good. The `eqeqeq` rule might be worth a look too.

Reply:
Yes, I didn't include it because when I started I didn't always use `===` and I'm concerned that changing everything to it now might create subtle bugs. New code should indeed use strict equality/inequality. Might still be worth reviewing at some point if we can enable it though.

Reply:
So no explicit return statement is bad style. Got it.